docs/examples/agrivoltaics/plot_diffuse_PAR_Spitters_relationship.py

```diff
@@ -0,0 +1,145 @@
+"""
+Calculating the diffuse PAR using Spitter's relationship
+=========================================================
+
+This example demonstrates how to calculate the diffuse photosynthetically
+active radiation (PAR) from total PAR using Spitter's relationship.
```

Comment:
```suggestion
active radiation (PAR) from total PAR using Spitter's relationship.
```
I think this is wrong. Shouldn't it be "from broadband diffuse fraction"?